wikimedia/wikipedia-ios
# [ALT TEXT] Add Onboarding

**Phabricator:** https://phabricator.wikimedia.org/T370315

### Notes
This PR uses our existing component to present alt text onboarding when entering the flow. Slight tweaks were made to the component to add the share icon and scroll indicator flashes.

### Test Steps
_Testing image recs flow_
1. Fresh install app. Enable feature flags.
2. Enter image recommendations flow. Publish edit.
3. Alt text prompt should appear. After tapping "Add", onboarding should appear.
4. Tapping "View examples" should kick you out to Safari.
5. Go back to the app and tap "Continue". App should dismiss onboarding and push to the Article + alt text input half sheet view.

_Testing article flow_

6. Delete app, fresh install again. Enable feature flags. Go to an article with an image on Test Wiki and publish an edit.
7. Alt text prompt should appear. After tapping "Add", onboarding should appear. Swipe to dismiss onboarding.
8. App should push on the Article + alt text input half sheet view.
